index.html

```diff
@@ -22,23 +22,43 @@
           src="images/image_geometry_1(1).svg"
           alt="My portfolio's logo"
       /></a>
-      <nav id="menuimage">
-        <img src="./images/ic_menu.svg" alt="menu icon" />
+      <nav>
+        <!-- Desktop nav-link starts-->
+        <ul id="header-nav">
+          <li><a href="index.html">Hello</a></li>
+          <li><a href="index.html">Portfolio</a></li>
+          <li><a href="index.html">About</a></li>
+          <li><a href="index.html">Contact</a></li>
+        </ul>
+        <!-- Desktop nav-link ends -->
+        <img src="./images/ic_menu.svg" alt="menu icon" id="menuimage" />
```

Comment:
- [ ] Your hamburger icon has disappeared from the mobile version as you can see below

![image](https://user-images.githubusercontent.com/90556221/166120744-05acd6c8-67c9-4981-a6e4-6f8cbaea712c.png)

Kindly make sure the mobile version stays the same even if you update the desktop version, such bugs are common occurrences during responsive development so please check it out an fix it.